metaworld/envs/mujoco/sawyer_xyz/sawyer_pick_place_wall_v2.py

```diff
@@ -0,0 +1,255 @@
+import numpy as np
+from gym.spaces import Box
+
+from metaworld.envs.env_util import get_asset_full_path
+from metaworld.envs.mujoco.sawyer_xyz.base import SawyerXYZEnv
+
+
+class SawyerPickPlaceWallEnvV2(SawyerXYZEnv):
+    """
+    Motivation for V2:
+        V1 was completely unsolvable because the observation didn't say where
+        to move after picking up the puck.
+    Changelog from V1 to V2:
+        - (6/24/20) Added a 3 element vector to the observation. This vector
```

Comment:
can you confirm/verify that you need all 3 elements (delta_x, delta_y, delta_z). Is it solveable with less information?